docs/repositories-configure/ignoring-files.md

```diff
@@ -10,16 +10,16 @@ To exclude files from your repository analysis open your repository **Settings**
 
 ![Ignored files](images/ignored-files.png)
 
-You can also ignore files using your own tool configuration files, although this depends on the option being supported by each tool.
+You can also ignore files using your own [tool configuration files](configuring-code-patterns.md#using-your-own-tool-configuration-files), although this depends on the option being supported by each tool.
```

Comment:
Since each tool uses a different config file format, we can't offer any examples here. I'm therefore just adding a link to point the user to the tool configuration section for more information.

Reply:
Yes, this is a good improvement. 👍